_includes/sidebar-data-v2.1.json

```diff
@@ -1538,6 +1538,12 @@
               "/${VERSION}/admin-ui-access-and-navigate.html"
             ]
           },
+          {
+            "title": "Enable User Authentication",
```

Comment:
I defer to you, but I still like "Secure the Admin UI" better than "Enable User Authentication". User auth is much broader than the admin ui.